lldb/source/Plugins/LanguageRuntime/Swift/SwiftMetadataCache.cpp

```diff
@@ -172,8 +172,9 @@ SwiftMetadataCache::generateHashTableBlob(
     auto &mangled_name = std::get<1>(pair);
     if (mangled_name.empty())
       continue;
-    auto offset = field_descriptor.getAddressData() -
-                  field_descriptors.startAddress().getAddressData();
+    auto offset =
+        field_descriptor.getAddressData().getAddressData() -
+        field_descriptors.startAddress().getAddressData().getAddressData();
```

Comment:
Are there any other options for the second `getAddressData()`? `getRawAddress()`?